packages/design/AddCommentButton/index.tsx

```diff
@@ -48,6 +48,12 @@ export const AddCommentButton = forwardRef<HTMLButtonElement, AddCommentButtonPr
       className
     );
 
+    useEffect(() => {
+      return () => {
+        clearTimeout(hoverRef.current);
+      };
+    }, []);
```

Comment:
Note for follow up: This code pre-dates `useDebouncedCallback` but we could probably simplify this a bit by using that hook here.